rollem.config.js

```diff
@@ -4,21 +4,7 @@ const babel = require('rollup-plugin-babel');
 const uglify = require('rollup-plugin-uglify');
 
 module.exports = [{
-  entry: 'blocking-elements.js',
-  format: 'umd',
-  dest: 'dist/blocking-elements.js',
-  plugins: [
-    resolve({
-      jsnext: true,
-      main: true
-    }),
-    commonjs(),
-    babel({
-      exclude: 'node_modules/**', // only transpile our source code
-    }),
-  ],
-}, {
-  entry: 'blocking-elements.js',
+  entry: 'dist/blocking-elements.js',
```

Comment:
The generated js does not have an IIFEE

Reply:
Fixed.

As discussed in chat, `/dist/blocking-elements.js` is now an ES module that exports a singleton instead of setting the `document.$blockingElements` global. The `.min.js` file does still set the global, and continues to be IIFE-guarded ES5 minified. Included usage instructions in the README.

Reply:
Uhm, I'm wondering if we should rather create a new `dist/blocking-elements-module.js` for the ES module case. The README.md is centered around the `document.$blockingElements` API, so probably `dist/blocking-elements.js` and `dist/blocking-elements.min.js` should differ only on minification (both should be IFEEd and set the `document.$blockingElements` global).

Reply:
Agreed, it is a little weird. Both files now set `document.$blockingElement`.